accession-commons-core/src/main/java/uk/ac/ebi/ampt2d/commons/accession/core/AccessioningService.java

```diff
@@ -46,7 +46,7 @@ public interface AccessioningService<MODEL, HASH, ACCESSION> {
      * @return List of wrapper objects containing the accessioned objects and their associated accessions and hashes
      * @throws AccessionCouldNotBeGeneratedException when accession could not be generated
      */
-    List<GetOrCreateAccessionWrapper<MODEL, HASH, ACCESSION>> getOrCreate(List<? extends MODEL> messages)
+    List<GetOrCreateAccessionWrapper<MODEL, HASH, ACCESSION>> getOrCreate(List<? extends MODEL> messages, String applicationInstanceId)
```

Comment:
Can you update the method comment to explain what is the `applicationInstanceId` and why we're passing it ?

Reply:
Also I'm wondering if we should avoid reusing the variable `applicationInstanceId` in case there are still beans that would populate it from the properties file.